src/services

```diff
@@ -1173,3 +1173,23 @@ p({
   "lazy": false,
   "serviceClass": "foam.nanos.medusa.MedusaStatusWatcher"
 })
+
+p({
+  "class":"foam.nanos.boot.NSpec",
+  "name":"approvalRequestClassificationDAO",
+  "serve":true,
+  "serviceScript": """
+    return new foam.dao.EasyDAO.Builder(x)
+      .setGuid(true)
+      .setAuthorize(false)
```

Comment:
Sure we don't want this authorized? I think we do.

Reply:
Removed and granted approvalrequestclassification.read.* to basicUser